src/i18n/strings/en_EN.json

```diff
@@ -2008,8 +2008,8 @@
             "send_message_start_dm": "Send your first message to invite <displayName/> to chat",
             "start_of_dm_history": "This is the beginning of your direct message history with <displayName/>.",
             "start_of_room": "This is the start of <roomName/>.",
-            "topic": "Topic: %(topic)s ",
-            "topic_edit": "Topic: %(topic)s (<a>edit</a>)",
+            "topic": "Topic: <a>%(topic)s</a>",
+            "topic_edit": "Topic: <b>%(topic)s</b> (<a>edit</a>)",
```

Comment:
I think if you're injecting HTML it should be `<topic/>`

Reply:
Sure but the translator needing to specify 2 nested placeholders is pointless.
They can use `<topic/>` and in the _t call you just pass `{ topic: () => <Linkify>{ topic.html }</Linkify> }` or whatever